src/features/surveys/models/SurveySubmissionsModel.ts

```diff
@@ -9,11 +9,11 @@ export default class SurveySubmissionsModel extends ModelBase {
   private _repo: SurveysRepo;
   private _surveyId: number;
 
-  constructor(env: Environment, orgId: number, surveyId: number) {
+  constructor(env: Environment, orgId: number, submissionId: number) {
     super();
 
     this._orgId = orgId;
-    this._surveyId = surveyId;
+    this._surveyId = submissionId;
```

Comment:
What's happening here? Why are you renaming this to `submissionId`?

Reply:
Ah this was when I mistakenly started using the SurveySubmissionsModel instead of the SurveySubmissionModel, it was accidentally committed.